en/docs/what-is-choreo.md

```diff
@@ -10,7 +10,8 @@ The following are some of the cutting-edge capabilities of Choreo:
 
  - Efficient and effective development:
     - **Code in any language**: Develop applications in your team's preferred programming language, harnessing existing skill sets to build powerful polyglot APIs, services, events, tasks, and more.
-    - **Native support for VSCode and GitHub**: Seamlessly collaborate and manage code with VSCode and GitHub in a feature-rich and visually-enhanced development environment.
+    - **Native support for VS Code**: Seamlessly collaborate and manage code using VS Code in a feature-rich, visually-enhanced development environment.
+    - **Git repository integration**: Effortlessly connect to Git-based platforms like GitHub, Bitbucket, or GitLab to develop components, either by linking an existing repository or starting with an empty one and committing your source code later.
```

Comment:
I dont think we need to add this part

"or starting with an empty one and committing your source code later."